ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -229,6 +309,15 @@ export default class StatusDropdown extends React.Component {
     const alertStatus = getStatus(alertSummary.status);
     const alertSummaryActiveTags = alertSummary.performance_tags || [];
 
+    const dueDate = this.renderDueDateCountdown(alertSummary.created);
```

Comment:
To me `dueDate` here is misleading since you are not returning the due date, but a tooltip text for each alert to triage.